notebooks/240-efficientdet-object-detection/240-efficientdet-object-detection.ipynb

```diff
@@ -0,0 +1,666 @@
+{
```

Comment:
<p>The images are too small. No one can read them, please enlarge :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>